packages/contracts-bedrock/deploy/000-ProxyAdmin.ts

```diff
@@ -0,0 +1,24 @@
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+
+import {
+  assertContractVariable,
+  deployAndVerifyAndThen,
+} from '../src/deploy-utils'
+
+const deployFn: DeployFunction = async (hre) => {
+  const { deployer } = await hre.getNamedAccounts()
+  await deployAndVerifyAndThen({
+    hre,
+    name: 'ProxyAdmin',
+    args: [deployer],
+    postDeployAction: async (contract) => {
+      // Owner is temporarily set to the deployer. We transfer ownership of the ProxyAdmin to the
```

Comment:
I see there is no legacy config of the `ProxyAdmin` which i suppose isn't needed here since this is a fresh deploy